src/Command/DispatchCommand.php

```diff
@@ -141,12 +141,18 @@ protected function execute(InputInterface $input, OutputInterface $output)
     private function updateRepositories(Package $package, array $projectConfig): void
     {
         $repositoryName = $this->getRepositoryName($package);
+        $lastVersion = $this->getLastPackageVersion($package);
+
         $branches = array_keys($projectConfig['branches']);
         $this->io->section('Repository');
 
         $repositoryInfo = $this->githubClient->repo()->show(static::GITHUB_GROUP, $repositoryName);
         $infoToUpdate = [
-            'homepage' => 'https://sonata-project.org/',
+            'homepage' => $lastVersion->getHomepage() ?: 'https://sonata-project.org/',
+            'description' => $lastVersion->isAbandoned()
+                ? '[Abandonned] '.$lastVersion->getDescription()
+                : $lastVersion->getDescription(),
```

Comment:
We could probably improve the composer.json tags and description, but this is better than before IMO.

Reply:
I would like to try to on admin bundle, @OskarStark may help me for that.

If it works as expected, I'll make PR to improve composer.json for every package.
And we'll need a new release for all of them.

Reply:
Will help but I am on holiday again 🤪

Back at next Monday 🥳

Reply:
@OskarStark I don't know well the dispatch command.

How can I tried it locally without trigger any change on sonata projects ?